tests/DCPS/WriteDataContainer/WriteDataContainerTest.cpp

```diff
@@ -350,6 +350,8 @@ int run_test(int argc, ACE_TCHAR *argv[])
 
   try
     {
+    DDS::DomainParticipantFactory_var dpf =
```

Comment:
The monitor member is for example zero at that point, not set to the default monitor impl